recipes/trisbm/build.sh

```diff
@@ -0,0 +1,4 @@
+rm -rf hSBM_topicmodel/
+git clone https://github.com/martingerlach/hSBM_Topicmodel
```

Comment:
Why is this done? This should likely be either in `sources` or applied as `patches` on top of the sources. As it stands the builds can't be reproducible as the git repo could change at any time.

Reply:
I moved them to sources